unikraft/app/application_options.go

```diff
@@ -210,3 +210,11 @@ func WithEnv(env map[string]string) ApplicationOption {
 		return nil
 	}
 }
+
+// WithLabels sets the list of labels.
+func WithLabels(labels map[string]string) ApplicationOption {
+	return func(ac *application) error {
+		ac.labels = labels
+		return nil
+	}
+}
```

Comment:
Can you also introduce a:
```go
func WithLabel(key, value string) ApplicationOption
```
Which sets the `value` for the `key`. (Remember to check if `ac.labels` is nil!)